pyscript.core/tests/integration/python/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <script src="mini-coi.js" scope="./"></script>
```

Comment:
as much as I think *mini-coi* validated in here *could* make some sense, I don't think we should test our code with a service worker ... our PyScript .com doesn't need or use one, so that test should likely be a part and very simple (if you can't change `window.document.body.textContent` from a worker, that's a failing Service Worker, nothing else literally changes).

I am saying this because behind the scene there is also sabayon SW approach but that's tested and code covered in sabayon already so we should provide "*production code*" (imho) ... your call (it wasn't needed before)